docs/sdkx-unity/troubleshooting-android.mdx

```diff
@@ -29,40 +29,45 @@ Helpshift Unity SDK X uses legacy android support libraries.
 If you want to use AndroidX libraries in your application please follow these steps to make Helpshift SDK compatible with AndroidX:
 
 1. We recommend to use the standalone jetifier tool mentioned [here](https://developer.android.com/studio/command-line/jetifier).
-1. To install Jetifier, [download the zip file](https://dl.google.com/dl/android/studio/jetifier-zips/1.0.0-beta07/jetifier-standalone.zip) and extract it. Your development environment must have Java version 1.8 installed.
-1. Use `standalone jetifier` tool to manually jetify the Helpshift library files:
+1. To install Jetifier, [download the zip file](https://dl.google.com/dl/android/studio/jetifier-zips/1.0.0-beta07/jetifier-standalone.zip) and extract it. Your development environment must have **Java version 1.8** installed.
```

Comment:
Please update the bullet points to be 1, 2, 3, 4...

I am aware that they render correctly in the final output, but it's a good idea to patch them up here.